interfaces/default/js/newznab.js

```diff
@@ -71,7 +71,48 @@ function search(query, catid, indexer) {
                         });
                         item.attr = attributes;
 
+			// First, assign download client category based on each individual usenet indexer
                         var clean_category = item.category.split(' > ')[0].toLowerCase().trim()
+			// Second, try to match with the newznab default to make it uniform across all indexers
+			// Maincategories
+			// 1 = console
+			// 2 = movies
+			// 3 = audio
+			// 4 = pc
+			// 5 = tv
+			// 6 = xxx
+			// 7 = other
+			var main_category = item.attr['category'].split("0",1)
+			if(main_category == 1) {
+			    var clean_category = "console"
```

Comment:
Can you remove all the var from the lines you added? Its already defined so there no need to do it again.